tests/CompaniesTest.php

```diff
@@ -104,6 +104,24 @@ public function it_should_throw_a_notfound_exception_when_client_receives_not_fo
         $custify->companies();
     }
 
+    /** @test */
+    public function it_should_throw_a_notfound_exception_when_client_receives_an_empty_company_while_getting_a_company_by_id()
```

Comment:
When getting a company that doesn’t exists?

Reply:
Same for people:

it_should_not_return_a_person_when_using_a_non_existing_user_id

People and company should essentially be the same.